prisma/migrations/20250328203730_wallet_badges/migration.sql

```diff
@@ -0,0 +1,95 @@
+ALTER TABLE "users"
+    ADD COLUMN     "hasRecvWallet" BOOLEAN NOT NULL DEFAULT false,
+    ADD COLUMN     "hasSendWallet" BOOLEAN NOT NULL DEFAULT false;
+
+ALTER TABLE "Streak"
+    ALTER COLUMN "startedAt" SET DATA TYPE TIMESTAMP(3),
+    ALTER COLUMN "endedAt" SET DATA TYPE TIMESTAMP(3);
```

Comment:
I had to change the types to support sending found/lost notifications for guns/horses at the same day.

I need to test that this didn't break cowboy hat streaks in some time-trippy way and if existing data is properly migrated to a timestamp at midnight in using the America/Chicago time zone.

Reply:
Tested, cowboy hat streaks continue to work as before